documentation/book/con-operator-configuration.adoc

```diff
@@ -0,0 +1,63 @@
+[id='operator-configuration-{context}']
+= Operator Configuration
+
+The operator itself can be configured through the following environment variables.
+
+[[STRIMZI_NAMESPACE]] `STRIMZI_NAMESPACE`:: Required. A comma-separated list of namespaces that the operator should
+operate in. The Cluster Operator deployment might use the https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api[{KubernetesName} Downward API]
```

Comment:
We should add this kubernetes link to the atrributes - also external links open in a new window - remember the ^

Reply:
http://strimzi.io/contributing/guide/#styling_links

Reply:
Procedure not concept